src/pages/singlevideo/SingleVideo.jsx

```diff
@@ -1,37 +1,45 @@
 import { useState, useEffect } from 'react'
 import { Navbar, Footer, VideoCard } from 'components'
-import { useParams, useNavigate, Link } from 'react-router-dom'
+import { useParams } from 'react-router-dom'
 import { videos } from 'backend/db/videos'
-// import { addPlaylist, deletePlaylistVideo, addVideoToPlaylist } from 'services'
```

Comment:
you can removed this code if not necessary

Reply:
sure